### ISSUE TITLE
logging.config=uri of config server??

### ISSUE THREAD

**Issue by 007anwar** [spring-cloud/spring-cloud-config#1673]
I have config server which uses serving plain text feature and returns config.xml file, but when I use it as **### (logging.config=http://localhost@localhost:8887/foo/development/master/configuration)** it gives ERROR Error parsing http://localhost:8887/foo/development/master/configuratio org.xml.sax.SAXParseException; systemId: http://localhost:8887/foo/development/master/configuratio; lineNumber: 11; columnNumber: 5; The element type "link" must be terminated by the matching end-tag "</link>".

Basically I want my config server to serve config.xml file and consumer application should pick that format for logging...

**Comment by spencergibb** [comment]
Are you missing the `n` in configuration?

**Comment by spring-cloud-issues** [comment]
If you would like us to look at this issue, please provide the requested information. If the information is not provided within the next 7 days this issue will be closed.
